### PR TITLE
clean-aeslc

### PR DESCRIPTION
**Pull request by sbmaruf**

### CODE REVIEW

File: promptsource/templates/aeslc/templates.yaml [hunk 1]
```diff
@@ -11,11 +11,13 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: context_question_2
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:19]:
```suggestion
    name: what_is_this_email_about
```

File: promptsource/templates/aeslc/templates.yaml [hunk 2]
```diff
@@ -27,11 +29,13 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: question_context_1
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:37]:
```suggestion
    name: what_is_the_subject_of_this_email
```

File: promptsource/templates/aeslc/templates.yaml [hunk 3]
```diff
@@ -43,27 +47,31 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: question_context_4
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:55]:
```suggestion
    name: the_text_below
```

File: promptsource/templates/aeslc/templates.yaml [hunk 4]
```diff
@@ -43,27 +47,31 @@ templates:
...
-    reference: ''
+    reference: Given a domain, ask a question
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:56]:
```suggestion
    reference: ''
```
what is "domain" here???

File: promptsource/templates/aeslc/templates.yaml [hunk 5]
```diff
@@ -43,27 +47,31 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: question_context_3
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:73]:
```suggestion
    name: generate_subject_line
```

File: promptsource/templates/aeslc/templates.yaml [hunk 6]
```diff
@@ -75,11 +83,13 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: question_context_2
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:91]:
```suggestion
    name: what_about
```

File: promptsource/templates/aeslc/templates.yaml [hunk 7]
```diff
@@ -91,27 +101,31 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: context_question_1
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:109]:
```suggestion
    name: what_subject_of_email
```

File: promptsource/templates/aeslc/templates.yaml [hunk 8]
```diff
@@ -91,27 +101,31 @@ templates:
...
       {{ subject_line }}'
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: context_question_3
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:127]:
```suggestion
    name: generate_subject
```

File: promptsource/templates/aeslc/templates.yaml [hunk 9]
```diff
@@ -124,8 +138,10 @@ templates:
...
       {{ subject_line }} '
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: false
+      metrics:
+      - ROUGE
+      - Other
       original_task: true
     name: context_question_4
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:146]:
```suggestion
    name: what_topic
```

File: promptsource/templates/aeslc/templates.yaml [hunk 10]
```diff
@@ -124,8 +138,10 @@ templates:
...
-    reference: ''
+    reference: Given a domain, ask a question
```
Comment (VictorSanh) [inline promptsource/templates/aeslc/templates.yaml:147]:
```suggestion
    reference: ''
```

**Review by VictorSanh — APPROVED** [review]
made a few fixes (change name for something more descriptive, what is domain???)

**Comment by VictorSanh** [comment]
thanks for working on this @sbmaruf 
merging now

**Comment by sbmaruf** [comment]
Thanks a lot @VictorSanh for making the corrections.

**Pull request merged**